docs/migration/4_0.md

```diff
@@ -719,6 +719,64 @@ The following properties where removed in 4.0:
 - Change detection strategy for this component was set to `OnPush`.
 - Spinner component was added to the template which relys on `isUpdating$` property.
 
+### LogoutGuard
+
+`AuthRedirectService` is a new, required constructor dependency.
+
+### (a11y) Screen Reader Improvements: Headings
+- Most of these changes affect only headings on screen reader elements dialog.
```

Comment:
it shouldn't be a seperated category. Changes below are just component changes. This header can be removed